src/vtcomposite.cpp

```diff
@@ -826,104 +824,130 @@ struct LocalizeWorker : Napi::AsyncWorker
 Napi::Value localize(Napi::CallbackInfo const& info)
```

Comment:
As an aside, I wonder how much time is taken by validating and processing the arguments. I don't expect you to address that in this PR though, just thinking aloud.